### PR TITLE
align routes

### PR DESCRIPTION
**Pull request by Tomology**
I have changed the following FE routes:

**dashboard/settingsPages** --> **settings/settingsPages** (I have also renamed the folder name from dashboard to settings).

**dashboard/profile** --> **/profile** (will be changed to /profile/id once integrated).

**profile/lists** --> **/listings**

I have also renamed the **/settings/settings** page to be **settings/account** as this page will deal with account deletion and other account-based options.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -36,26 +36,22 @@ function App() {
             <Switch>
               <Route path="/signup" component={Signup} />
               <Route path="/login" component={Login} />
-              <PrivateRoute path="/profile/lists" component={ProfileListings} />
-              <PrivateRoute path="/" exact component={Profile} />
+              <PrivateRoute path="/listings" component={ProfileListings} />
+              <PrivateRoute path="/profile" exact component={Profile} />
```
Comment (dproc96) [inline client/src/App.js]: Why is this here twice?

**Review by dproc96 — APPROVED** [review]
Noticed one thing, otherwise looks good! Be careful resolving the conflicts though, I find they can be tricky with renamed file paths sometimes

File: client/src/App.js (hunk 1 above)
Comment (Tomology) [reply client/src/App.js]: removed

**Comment by dproc96** [comment]
@Tomology What's the status on handling these conflicts?
